niri-config/src/lib.rs

```diff
@@ -335,6 +337,7 @@ impl Default for Output {
             position: None,
             mode: None,
             variable_refresh_rate: false,
+            background_color: Color::from_rgba8_unpremul(51, 51, 51, 255),
```

Comment:
I feel like since we use this default color both here and in niri.rs, we should move the `CLEAR_COLOR` constant here and call it something like `DEFAULT_BACKGROUND_COLOR`.